runtime/pack/dist/opt/comment/doc/comment.txt

```diff
@@ -32,10 +32,19 @@ back to `gcc` otherwise, add the following mapping to your vimrc: >
 Note: using `gC` may not always result in valid comment markers depending on
 the language used.
 
+Additionally, the plugin defines a comment text-object which requires syntax
+highlighting to be enabled.
+						     *v_ac* *ac*
+ac		"a comment", select current or next comment.
+		Leading and trailing white space is included.
+		Trailing newlines are included too.
+						     *v_ic* *ic*
+ic		"inner comment", select current or next comment.
+
 This plugin uses the buffer-local 'commentstring' option value to add or remove
 comment markers to the selected lines.  Whether it will comment or un-comment
-depends on the first line of the range of lines to act upon.  When it matches
-a comment marker, the line will be un-commented, if it doesn't, the line will
+depends on the range of lines to act upon.  When all of the lines in range
+have comment markers, all lines will be un-commented, if it doesn't, the lines will
```

Comment:
My eyeball suggests this line now overruns the margin, but I didn't measure precisely.

Reply:
eagle eye!

fixed.